### DISCUSSION TITLE
Add POST method to API endpoint ​/api​/ruler​/grafana/api/v1/rules endpoint

### DISCUSSION THREAD

**Discussion by dezpriest** [grafana/grafana#37671] [Alerting]
**What would you like to be added:**
POST method to the alerting API endpoint ​`/api​/ruler​/grafana/api/v1/rules`
It would accept multiple rules from multiple Namespaces and apply them to the Grafana.

**Why is this needed:**

Right now we can gather all configured alerts by sending **GET** request to the endpoint ​`/api​/ruler​/grafana/api/v1/rules`.
But if we want to add new alerts - we need to use another endpoint - **POST**
​`/api​/ruler​/{Recipient}​/api​/v1​/rules​/{Namespace}`.

So, if we would like to gather all alerts from one instance of Grafana and apply them to another Grafana instance -  we will need to do following:
- get the alerts from 1st endpoint
- parse the output JSON and split it into multiple JSONs (per Namespace)
- post these JSONs to 2nd endpoint

It would be great to just simply POST all alerts using the same endpoint `/api​/ruler​/grafana/api/v1/rules` that we used to GET them.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
